### ISSUE TITLE
Auto DB tables generation

### ISSUE THREAD

**Issue by kbariotis** [marmelab/microrest.php#9]
It would be nice if you could also generate DB tables based on the RAML file.

Great job! Cheers

**Comment by alexisjanvier** [comment]
That's right, it's a good feature, we intend to develop. Perhaps for Sqlite in a first time ?

Thank you for your encouragements.

**Comment by kbariotis** [comment]
Yeap Sqlite would be a great PoC. 

Maybe i could find some time next week and get it started. Would that be desired?
